llvm/lib/Target/X86/X86InstrSSE.td

```diff
@@ -7931,6 +7927,14 @@ let Predicates = [HasAVX2, NoVLX] in {
   defm : vextract_lowering<"VEXTRACTI128", v32i8,  v16i8>;
 }
 
+let Predicates = [HasAVXNECONVERT, NoVLX] in {
+  def : Pat<(v16bf16 (X86SubVBroadcastld128 addr:$src)),
```

Comment:
This file seems to have a good organization on categories of broadcast, insert and extract. Why put these three together?

Reply:
In that way, we need to check Predicates each time. I'm fine with either way. I can change it if you prefer.

Reply:
Yes, I prefer to the old way.

Reply:
Done.